libfreerdp/gdi/gfx.c

```diff
@@ -1165,13 +1303,15 @@ void gdi_graphics_pipeline_init(rdpGdi* gdi, RdpgfxClientContext* gfx)
 	gfx->MapSurfaceToOutput = gdi_MapSurfaceToOutput;
 	gfx->MapSurfaceToWindow = gdi_MapSurfaceToWindow;
 	gfx->UpdateSurfaces = gdi_UpdateSurfaces;
+	gfx->mux = CreateMutexA(NULL, FALSE, "gfx-mutex");
 	PROFILER_CREATE(gfx->SurfaceProfiler, "GFX-PROFILER")
```

Comment:
value must be checked.
why creating a named mutex ?

Reply:
Isn't a `CRITICAL_SECTION` ok ?

Reply:
@hardening already done ;)